### ISSUE TITLE
Blank element bug

### ISSUE THREAD

**Issue by LedAlex** [mltframework/shotcut#955]
Shotcut version 20.10.31

If you delete (usually accidentally) original clips from transition, revert them back and save project. Project becomes broken and transition too. There is no way to fix it legally only with some magic.

Bug exists in all Shotcut versions for long time.
I already have had several seriously broken projects.
And only now I've figured out how to reproduce it 100%.

[Shotcut Blank element bug](https://youtu.be/I1mOWgy5YDk)

**Comment by ddennedy** [comment]
Yes, it is a bad bug, but there is a much easier way to repair the project: click OK on the Missing Files dialog and simply delete the transition that shows INVALID.
